unity-client/Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/NFTShape/NFTShapeAssetsType.cs

```diff
@@ -0,0 +1,180 @@
+﻿using System;
+using System.Collections;
+using DCL;
+using DCL.Controllers.Gif;
+using UnityEngine;
+
+internal interface INFTAsset : IDisposable
+{
+    bool isGif { get; }
+    bool isHQ { get; }
+    DCL.ITexture previewAsset { get; }
+    DCL.ITexture hqAsset { get; }
+    Action<Texture2D> UpdateTextureCallback { set; }
+    void FetchAndSetHQAsset(string url, Action onSuccess, Action onFail);
+    void RestorePreviewAsset();
+}
```

Comment:
yeah, you are totally right, at the beginning I was using that boolean for other stuffs to avoid checking type in runtime but later I polish the code better and forgot about it :P